packages/react-juce/src/components/Canvas.ts

```diff
@@ -59,13 +60,15 @@ export function bindCanvasContextProperties(ctx: any) {
 
 export interface CanvasProps {
   onMeasure?: (e: any) => void;
-  onDraw: (ctx: CanvasRenderingContext2D) => void;
+  onDraw: (ctx: CanvasRenderingContext2D, sizeState: object) => void;
```

Comment:
This is probably not a necessary addition. 

We should be able to pass width/height to draw routines through other means. For example in your `App` component's `drawingCanvasDraw(ctx, sizeState)` function, instead of passing sizeState to `onDraw` you would add `width` and `height` to `App`'s internal state and then inside `drawingCanvasDraw` refer to `this.state.width` etc. The idea being that `drawingCanvasDraw` is a member function of your `App` component class bound to `this`.